blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,68 @@
+
+Read on to discover the exciting new features and how they can benefit you!
+
+## Key Features, Contributions, and Decprecations
+
+Key features in this release include:
+- Cosign Verifier Enhancements 
+- Kubernetes Multi-Tenancy Support
+- Key Management Provider
+
+Contributions from the community:
+- [@duffney](https://github.com/duffney) made their first contribution: [support plugin logging #1254](https://github.com/deislabs/ratify/pull/1254)
+- [@mannbiher](https://github.com/mannbiher) made their first contribution: [fix: Set IdleTimeout for http.Server #1418](https://github.com/deislabs/ratify/pull/1418)
+
+Deprecations:
+- `CertificateStore` is deprecated in favor of `KeyManagementProvider` (KMP). Please migrate to `KeyManagementProvider` by following guide [here](https://ratify.dev/docs/reference/custom%20resources/key-management-providers#migrating-from-certificatestore-to-kmp). Support will be removed in Ratify v2.0.0
+
+## A brand new Cosign experience! 
+
+Starting in v1.2.0, Ratify now has more comprehensive Cosign verification capabilities. 
+
+Verification of Cosign signatures is a critical validation. As such, the Cosign verifier is now a built-in verifier along with the Notation verifier. Beyond first class support, the Cosign verifier introduces a `Trust Policy` which binds a particular set of verification settings to a certain registry scope. Now, it's possible to specify multiple trust policies based on your registry, namespace, and/or image. 
+
+Cosign verification also includes a robust key experience that integrates directly with KMP resources. Expanded capabilities include: multiple key support from KMP or filesystem and RSA + elliptical keys across multiple sizes.
+
+Watch this short demo to see the Cosign Verifier in action. Or check out the [documentation](https://ratify.dev/docs/reference/verifiers/cosign) for more information.
+
+[![asciicast](https://asciinema.org/a/658139.svg)](https://asciinema.org/a/658139)
+
+## Support for Multi-Tenancy in Kubernetes
+
+Ratify now supports multi-tenancy, enabling organizations to share a single cluster among multiple teams by allowing users to define resources within a namespace scope. 
+
+This enhancement is seamlessly integrated, as users implicitly opt into multi-tenancy by applying namespace-specific resources such as NamespacedPolicy and NamespacedVerifier, without the need for an explicit feature flag. 
+
+Currenlty multi-tenancy supports the following resources: `Referrer Store`, `Verifier`, and `Key Management Provider`, allowing these to be defined and managed within specific namespaces. 
```

Comment:
`Policy` is also supported in multi-tenancy scenario.

Reply:
I'll get that added. Thank you for calling that out @binbin-li